src/app/fsat/calculate-pressures/calculate-flow-pressure/calculate-flow-pressure.component.ts

```diff
@@ -35,7 +35,8 @@ export class CalculateFlowPressureComponent implements OnInit {
 
   ngOnInit() {
     this.fanAnalysisService.inAssessmentModal = true;
-    this.fanAnalysisService.inputData = this.fanAnalysisService.getFlowPressureModalDefaults();
+    this.fanAnalysisService.inputData = this.fanAnalysisService.getExampleData();
+    //this.fanAnalysisService.inputData = this.fanAnalysisService.getFlowPressureModalDefaults();
```

Comment:
getExampleData() was just for testing. We'll need to flip this back to using getFlowPressureModalDEfaults().